modules/profile/variables.tf

```diff
@@ -0,0 +1,41 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "instance_id" {
+  type        = string
+  description = "The ID of the SCC instance in a particular region."
+}
+
+variable "profile_name" {
+  type        = string
+  description = "The profile name. Constraints: The maximum length is `64` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "profile_description" {
+  type        = string
+  description = "The profile description. Constraints: The maximum length is `256` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
```

Comment:
I would remove the `Constraints` part - this might become out of sync with the provider